### PR TITLE
Milestone 4: Calculations 

### PR DESCRIPTION
**Pull request by SarvarKh**
## In this milestone, I have:
- [x] prepared the directory structure for logics;
- [x] implemented calculate.js and operate.js files;
- [x] fixed all linter errors;

### CODE REVIEW

File: src/logic/calculate.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+/* eslint no-unused-vars: 0 */
```
Comment (afanimartin) [inline src/logic/calculate.js]: - Fix that error. It's easy and straightforward.

File: src/logic/calculate.js [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+    total *= (-1);
+    next *= (-1);
+  } else if (buttonName === 'AC') {
+    total = 0;
+    next = 0;
+    operation = 0;
+  } else if (buttonName === '=') {
+    total = operate(total, next, operation);
+    next = '';
+    operation = null;
```
Comment (afanimartin) [inline src/logic/calculate.js]: - Add implementation from button name `.` for performing calculations with decimal points

File: src/logic/operate.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+import Big from 'big.js';
+
+const operate = (numberOne, numberTwo, operation) => {
+  const bigNumOne = new Big(numberOne);
+  const bigNumTwo = new Big(numberTwo);
+  let result;
+
+  if (operation === '-') {
+    result = bigNumOne - bigNumTwo;
```
Comment (afanimartin) [inline src/logic/operate.js]: - Use `bigNumOne.minus(bigNumTwo)`. This is a function that already exists and it makes your code more readable.

File: src/logic/operate.js [hunk 4]
```diff
@@ -0,0 +1,23 @@
...
+  } if (operation === '+') {
+    result = bigNumOne + bigNumTwo;
```
Comment (afanimartin) [inline src/logic/operate.js]: - Use `bigNumOne.plus(bigNumTwo)`

File: src/logic/operate.js [hunk 5]
```diff
@@ -0,0 +1,23 @@
...
+  } if (operation === 'X') {
+    result = bigNumOne * bigNumTwo;
```
Comment (afanimartin) [inline src/logic/operate.js]: - Use `bigNumOne.times(bigNumTwo)`

File: src/logic/operate.js [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+  } if (operation === '÷') {
+    result = bigNumOne / bigNumTwo;
```
Comment (afanimartin) [inline src/logic/operate.js]: - Use `bigNumOne.div(bigNumTwo)`

File: src/logic/operate.js [hunk 7]
```diff
@@ -0,0 +1,23 @@
...
+  } if (operation === '%') {
+    result = bigNumOne / 100;
```
Comment (afanimartin) [inline src/logic/operate.js]: - Use `bigNumOne.mod(bigNumTwo)`

**Review by afanimartin — CHANGES_REQUESTED** [review]
## Changes Required 🙏🏿 🙏🏿 🙏🏿 
Nice work on this milestone. Most requirements have been taken care of.

### Highlights 🌵 🌵 🌵 
- app is deployed
- app starts correctly with `npm start` command
- unnecessary files and folders removed

### Changes 🔕 🔕 🔕 
- see comments for suggestions


### Martin Afani
#### Slack: whiz25

File: src/logic/calculate.js (hunk 1 above)
Comment (SarvarKh) [reply src/logic/calculate.js]: Fixed

File: src/logic/calculate.js (hunk 2 above)
Comment (SarvarKh) [reply src/logic/calculate.js]: Added

File: src/logic/operate.js (hunk 3 above)
Comment (SarvarKh) [reply src/logic/operate.js]: Implemented

File: src/logic/operate.js (hunk 4 above)
Comment (SarvarKh) [reply src/logic/operate.js]: Implemented

File: src/logic/operate.js (hunk 5 above)
Comment (SarvarKh) [reply src/logic/operate.js]: Implemented

File: src/logic/operate.js (hunk 6 above)
Comment (SarvarKh) [reply src/logic/operate.js]: Implemented

File: src/logic/operate.js (hunk 7 above)
Comment (SarvarKh) [reply src/logic/operate.js]: Implemented

**Review by NtwaliHeritier — APPROVED** [review]
### **STATUS: APPROVED👍️👍️👍️**

Hello @SarvarKh 
Great work making the request changes.👏️👏️

You can merge.👍️👍️

Happy coding💻️💻️

**Pull request merged**